src/test/vsphere-csi-driver/templates/controller.yaml

```diff
@@ -0,0 +1,240 @@
+          args:
+          {{- if (gt (.Values.replicas | int) 1) }}
+            - "--leader-election"
+          {{- end }}
+            - "--fss-name=internal-feature-states.csi.vsphere.vmware.com"
+            - "--fss-namespace=$(CSI_NAMESPACE)"
+          imagePullPolicy: "Always"
+          ports:
+            - containerPort: 2113
+              name: prometheus
+              protocol: TCP
+          env:
+            - name: FULL_SYNC_INTERVAL_MINUTES
+              value: "30"
+            - name: VSPHERE_CSI_CONFIG
+              value: "/etc/cloud/csi.conf"
+            - name: LOGGER_LEVEL
+              value: "PRODUCTION" # Options: DEVELOPMENT, PRODUCTION
+            - name: INCLUSTER_CLIENT_QPS
+              value: "100"
+            - name: INCLUSTER_CLIENT_BURST
+              value: "100"
+            - name: CSI_NAMESPACE
+              valueFrom:
+                fieldRef:
+                  fieldPath: metadata.namespace
+          volumeMounts:
+            - mountPath: /etc/cloud
+              name: vsphere-config-volume
+              readOnly: true
+        - name: csi-provisioner
+          image: {{ .Values.provisioner.image.repository }}:{{ .Values.provisioner.image.tag }}
+          args:
+            - "--v={{ .Values.provisioner.verbose }}"
+            - "--csi-address=$(ADDRESS)"
+            - "--metrics-address=:22011"
+            - "--timeout={{ .Values.provisioner.timeout }}"
+            - "--retry-interval-start={{ .Values.provisioner.retryIntervalStart }}"
+            - "--retry-interval-max={{ .Values.provisioner.retryIntervalMax }}"
+            - "--worker-threads={{ .Values.provisioner.workerThreads }}"
+            # needed only for topology aware setup
+            {{- if .Values.enableTopologyZone }}
+            - "--feature-gates=Topology=true"
+            - "--strict-topology"
+            {{- end}}
+            - "--kube-api-qps=100"
+            - "--kube-api-burst=100"
+            {{- if (gt (.Values.replicas | int) 1) }}
+            - "--enable-leader-election"
+            - "--leader-election-type=leases"
+            {{- end }}
+            - "--default-fstype=ext4"
+          env:
+            - name: ADDRESS
+              value: /csi/csi.sock
+          volumeMounts:
+            - mountPath: /csi
+              name: socket-dir
+      volumes:
+      - name: vsphere-config-volume
+        secret:
+          secretName: csi-config-secret
+      - name: socket-dir
+        hostPath:
+          path: /var/lib/csi/sockets/pluginproxy/csi.vsphere.vmware.com
+          type: DirectoryOrCreate
+---
+apiVersion: v1
+data:
+  "volume-extend": "{{ .Values.storageclass.allowVolumeExpansion }}"
+  "online-volume-extend": "{{ .Values.storageclass.allowVolumeExpansion }}"
+  "volume-health": "true"
+  "csi-migration": "false"
+  "csi-auth-check": "true"
+  "online-volume-extend": "true"
```

Comment:
the field online-volume-extend already defined at line 204, you can delete line 208.